src/test/java/org/template/recommendation/PreparatorTest.java

```diff
@@ -1,8 +1,9 @@
 package org.template.recommendation;
 
 
+import javafx.util.Pair;
```

Comment:
With all the imports, don't forget to do Optimize Imports by right clicking on the project!